app/views/govuk_publishing_components/components/_reorderable_list.html.erb

```diff
@@ -0,0 +1,44 @@
+<%
+  items ||= []
+  input_name ||= "ordering"
+  data_attributes ||= {}
+  data_attributes[:module] = "reorderable-list"
+%>
+
+<%= tag.ol class: "gem-c-reorderable-list", data: data_attributes do %>
+  <% items.each_with_index do |item, index| %>
+    <%= tag.li class: "gem-c-reorderable-list__item" do %>
+      <%= tag.div class: "gem-c-reorderable-list__wrapper" do %>
+        <%= tag.div class: "gem-c-reorderable-list__content" do %>
+          <%= tag.p item[:title], class: "gem-c-reorderable-list__title" %>
+          <%= tag.p(item[:description], class: "gem-c-reorderable-list__description") if item[:description].present? %>
+        <% end %>
+        <%= tag.div class: "gem-c-reorderable-list__actions" do %>
+          <%= render "govuk_publishing_components/components/input", {
+            label: {
+              text: sanitize("Position<span class='govuk-visually-hidden'> for #{item[:title]}</span>")
```

Comment:
I recall that we did the work that we shouldn't need sanitize here in collections-publisher: https://github.com/alphagov/collections-publisher/blob/ca14ae8058f4115fe60eaefba8e4102a2c8ae19e/app/views/components/_reorderable_list.html.erb#L17-L19

Though I saw it crept back in a couple of lines later 🙈 . I imagine we can still get rid of it with using capture.

Reply:
done